test/browser_tests/cucumber/features/suites/default/global-search.feature

```diff
@@ -17,11 +17,9 @@ Feature: Global Search
     Then the search molecule shouldn't have a search trigger
     And it should have search input content
     And it should focus the search input field
-    And it shouldn't have a clear button label
```

Comment:
The step-definitions would need to be removed too, right? https://github.com/cfpb/cfgov-refresh/blob/master/test/browser_tests/cucumber/step_definitions/global-search.js#L75

Reply:
Whoops, good catch